perceval/utils/logging.py

```diff
@@ -0,0 +1,249 @@
+# MIT License
```

Comment:
This file might need to be separated in at least 2 or 3. Why don't you put all the logger code in a new sub-package (e.g. in perceval/utils/logger)?